docs/CHANGELOG/changelog-0.1.x.md

```diff
@@ -0,0 +1,20 @@
+# v0.1.0
+
+First official release of Claudie
```

Comment:
```diff
-First
+The first
```

Also, should we note somewhere around here that this is beta-quality software and breaking changes are expected?

Reply:
>Also, should we note somewhere around here that this is beta-quality software and breaking changes are expected?

I am not sure about this, let's ask others what they think about this comment. From my understanding, anything that is not on at least version `v1.0` is expected to change and does not support full backwards compatibility.

Reply:
I agree with @MiroslavRepka

Reply:
correct, I agree with what you said, as well @MiroslavRepka
however, not everyone might think of it as we do, therefore explicitly stating that is IMO better.